testing/docs/gettingStarted.md

```diff
@@ -0,0 +1,38 @@
+## Testing locally with Cypress
+
+_Before running these steps please visit the [testing.md](../../docs/testing.md) for a better understand of the overall testing coverage and approach for the project_
+
+__Please note all scripts/commands must be run in the `testing` folder__
+
+See [Installing Cypress Documentation](https://docs.cypress.io/guides/getting-started/installing-cypress#System-requirements) to get Cypress set up on local. If this is your first time using Cypress you will need to run the install script in your command line.
+
+### Command line for download
+```
+npm install cypress --save-dev
+```
```

Comment:
For the intialization in your local I believe you just need to run `npm install` cypress is already a listed dependency and will be installed along with the other dependencies.

Reply:
updated to just list `npm install`